examples/build.rs

```diff
@@ -30,4 +29,24 @@ fn main() {
             &["proto/googleapis"],
         )
         .unwrap();
+
+    build_json_codec_service();
+}
+
+fn build_json_codec_service() {
+    let greeter_service = tonic_build::rust::Service::builder()
+        .name("Greeter")
+        .package("json.helloworld")
+        .method(
+            tonic_build::rust::Method::builder()
+                .name("say_hello")
+                .route_name("SayHello")
+                .input_type("crate::HelloRequest")
+                .output_type("crate::HelloResponse")
+                .codec_path("crate::JsonCodec")
```

Comment:
should we make these `crate::common` to be more clear this is tying in the types from that module file? My concern is that this build.rs is quite far from that code and crate seems a bit less obvious.